package.js

```diff
@@ -1,19 +1,30 @@
 Package.describe({
   name: 'coagmano:stylus',
-  version: '2.0.0',
-  summary: 'Stylus plugin with plugins from mquandalle:stylus. Compatible with Meteor 1.4 and \'ecmascript\'',
+  version: '2.0.3',
+  summary: 'Stylus plugin with plugins from mquandalle:stylus. Compatible with Meteor 3.0',
   git: 'https://github.com/coagmano/meteor-stylus.git',
   readme: 'README.md'
 });
 
 Package.registerBuildPlugin({
   name: 'compileStylusBatch',
-  use: ['ecmascript@0.12.0', 'caching-compiler@1.2.0'],
+  use: ['ecmascript', 'caching-compiler'],
```

Comment:
We need the versions here to publish. Since this is for Meteor 3 we will have to put in the ones that will be part of the final release or do beta releases with the ones in those.

Reply:
With that though, we might consider v3 altogether. In the meantime I will publish this as compatible with Meteor 2.

Reply:
This makes sense. I would also say, don't bother with intermediary beta versions.